src/USER-MISC/fix_imd.cpp

```diff
@@ -405,7 +405,7 @@ static int imd_recv_fcoords(void *, int32, float *);
 /** Prepare IMD data packet header */
 static void imd_fill_header(IMDheader *header, IMDType type, int32 length);
 /** Write data to socket */
-static int32 imd_writen(void *s, const char *ptr, int32 n);
+static int32 imd_written(void *s, const char *ptr, int32 n);
```

Comment:
the changes to this file here are wrong.
the name of the function is supposed to be imd_writen (i.e. write n bytes over IMD connection).